### PR TITLE
getTotalCount method raises Exception for MDR list (Release 5.0.0)

### PR DESCRIPTION
**Pull request by mohsin-plivo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [5.0.0](https://github.com/plivo/plivo-java/tree/v5.0.0) (2021-07-05)
+- **BREAKING**: getTotalCount() method removed for list MDR.
+- Moved get() method to PowerPackLister
```
Comment (ppai-plivo) [inline CHANGELOG.md:5]: The moving of get() part - Should this be communicated in changelog? IOW, will this change break existing users? If not, we can remove it.

File: src/main/java/com/plivo/api/models/base/Meta.java [hunk 2]
```diff
@@ -24,7 +26,8 @@ public String getPrevious() {
     return previous;
   }
 
-  public Long getTotalCount() {
+  public Long getTotalCount() throws NoSuchMethodException {
```
Comment (ppai-plivo) [inline src/main/java/com/plivo/api/models/base/Meta.java:29]: On second thought, we're calling this out in release notes that the method is removed. What's your thought on actually removing this method instead?

cc @neel-plivo

**Review by narayana-plivo — APPROVED** [review]

File: src/main/java/com/plivo/api/models/base/Meta.java (hunk 2 above)
Comment (ppai-plivo) [reply src/main/java/com/plivo/api/models/base/Meta.java:29]: Just realised that this is the method of the base class.

**Review by ppai-plivo — APPROVED** [review]
LGTM 👍🏽

File: src/main/java/com/plivo/api/models/base/Meta.java (hunk 2 above)
Comment (mohsin-plivo) [reply src/main/java/com/plivo/api/models/base/Meta.java:29]: The other teams like voice, powerpack, etc might still be using the totalCount field so the method can't be removed from the base class. As for the child class, we couldn't find any provision for removing the method. The closest thing was to throw `NoSucMethodException`

**Pull request merged**